roles/ceph-rgw/tasks/main.yml

```
@@ -9,7 +9,9 @@
 - name: rgw pool creation tasks
   include_tasks: rgw_create_pools.yml
   run_once: true
-  when: rgw_create_pools is defined
+  when:
+    - rgw_create_pools is defined
+    - not rolling_update | default(False) | bool
```

```suggestion
    - not rolling_update | bool
```

It's just a minor thing since the rolling_update variable value is already set at false by default.